<ui version="4.0" >
 <class>MediaflyPlayqueue</class>
 <widget class="QWidget" name="MediaflyPlayqueue" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>493</height>
   </rect>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="6" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Mediafly Playqueue</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="6" >
    <widget class="mf::List" name="m_listView" >
     <property name="font" >
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>></string>
       </property>
       <property name="indent" >
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_songName" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Current playing song's name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="6" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_songPosition" >
       <property name="font" >
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_songLength" >
       <property name="font" >
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Song's length</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="4" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" >
    <widget class="QLabel" name="label_4" >
     <property name="font" >
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5" >
    <widget class="QLabel" name="label_2" >
     <property name="font" >
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>?</string>
     </property>
     <property name="margin" >
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="5" column="3" >
    <widget class="QLabel" name="label_5" >
     <property name="font" >
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Now Playing</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mf::List</class>
   <extends>QListView</extends>
   <header>List.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
